### DISCUSSION TITLE
How-dy? 🤠

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#960] [Galaxy Trivia] [answered]
A cowboy rides into town on Friday. He stays three days, then rides out of town on Friday.

**Comment by Carol42** [comment, marked as answer]
The horse's name is Friday :horse:

**Comment by thuongtruong1009** [comment]
Howdy! It seems like you've presented a riddle. The answer to this riddle is that the cowboy's horse is named Friday. So, when the cowboy says he rides into town on Friday, it means he is riding his horse named Friday. Similarly, when he says he rides out of town on Friday, it means he is still riding his horse named Friday.
